index.html

```diff
@@ -38,15 +38,15 @@ <h1>heading1</h1>
                 <p>I am Zhou Yang from China. I like computer ga mes and football very much.</p>
                 <p>My favorite football club is Manchester City, because their football is very beautiful.</p>
             </section>
-            <section class="bio">
+            <section class="favourites">
```

Comment:
I changed the classes on these two sections because they are not the 'bio' sections.